lang/python/python3-pyroute2/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-pyroute2
+PKG_VERSION:=0.5.5
+PKG_RELEASE:=1
+
+PKG_SOURCE:=pyroute2-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/pyroute2
+PKG_HASH:=ad679a91d453fe8426c4076d0da3a67265e5ccfe641879d75c9bc7660d075dfa
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/pyroute2-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Martin Matějek <martin.matejek@nic.cz>
+PKG_LICENSE:=GPL-2.0-or-later Apache-2.0
```

Comment:
The [license tag](https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#common-license-tags-short-list) for GPL v2 or later is `GPL-2.0+`.

Reply:
Oops, I wasn't aware `GPL-2.0-or-later` is the new identifier (#8709).

Reply:
I wasn't sure about it, but recently I saw discussion, where @diizzyy pointed it out and sent the link of the discussion to my colleague. I guess, we will see how my PR about new identifies will go. Don't worry, my packages still use the old identifiers. Once there upstream release new versions, I will update it or will do tree-wide commit. I haven't decided that.

But i2c-tools uses a correct SPDX tag and this PR as well. It was approved by both Python maintainers, I'm merging it.